airflow/models/dag.py

```diff
@@ -543,8 +543,13 @@ def get_doc_md(self, doc_md: str) -> str:
         if not doc_md.endswith('.md'):
             template = jinja2.Template(doc_md)
         else:
-            # template = env.loader.get_source(env, doc_md)[0]
-            template = env.get_template(doc_md)
+            try:
```

Comment:
Much better :)

Reply:
Yeah. I think failing DAG because of typo in doc was just a bit too hard on the user

Reply:
And it's perfectly OK to simply display the error as the documentation :)